flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/SqlITCase.scala

```diff
@@ -171,4 +183,79 @@ class SqlITCase extends StreamingMultipleProgramsTestBase {
     val expected = mutable.MutableList("Hello", "Hello world")
     assertEquals(expected.sorted, StreamITCase.testResults.sorted)
   }
+
+  @Test
+  def testUnboundPartitionedProcessingWindowWithRange(): Unit = {
+    val env = StreamExecutionEnvironment.getExecutionEnvironment
+    val tEnv = TableEnvironment.getTableEnvironment(env)
+    StreamITCase.testResults = mutable.MutableList()
+
+    val t1 = env.fromCollection(data).toTable(tEnv).as('a, 'b, 'c)
+
+    tEnv.registerTable("T1", t1)
+
+    //TODO ORDER BY will be fixed, once FLINK-5710 is solved
+    tEnv.registerFunction("ProcTime", ProcTime)
+    val sqlQuery = "SELECT c, count(a) OVER (PARTITION BY c ORDER BY ProcTime() RANGE " +
```

Comment:
Yes use `ORDER BY b` in query returns the same result. We only in the sql verification stage using `b` right?